### PR TITLE
libsquish 1.14

### PR DESCRIPTION
**Pull request by dholm**

### CODE REVIEW

File: Formula/libsquish.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cc").write <<-EOF.undent
```
Comment (DomT4) [inline Formula/libsquish.rb]: Minor nit, but we use `EOS.undent` and `EOS` mostly elsewhere so would be nice to be consistent here.

File: Formula/libsquish.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    #include <stdio.h>
```
Comment (DomT4) [inline Formula/libsquish.rb]: Can you indent this by two spaces? Thanks!

File: Formula/libsquish.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    #include <squish.h>
+    int main(void) {
+      printf("%d", GetStorageRequirements(640, 480, squish::kDxt1));
+      return 0;
+    }
+    EOF
+    system ENV.cxx, "-o", "test", "test.cc", "#{lib}/libsquish.a"
```
Comment (DomT4) [inline Formula/libsquish.rb]: `lib/"libsquish.a"` is a little more compact.

File: Formula/libsquish.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    assert_equal "153600", `./test`
```
Comment (DomT4) [inline Formula/libsquish.rb]:
```ruby
assert_equal "153600", shell_output("./test")
```

**Review by DomT4 — CHANGES_REQUESTED** [review]

**Comment by bfontaine** [comment]
@dholm Ping on @DomT4’s comments above?

**Comment by dholm** [comment]
@bfontaine Done!

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**